src/viur/core/bones/captcha.py

```diff
@@ -1,82 +1,130 @@
-import json
-import urllib.parse
-import urllib.request
+import logging
 import typing as t
 
+import requests
+
 from viur.core import conf, current
 from viur.core.bones.base import BaseBone, ReadFromClientError, ReadFromClientErrorSeverity
 
+if t.TYPE_CHECKING:
+    from viur.core.skeleton import SkeletonInstance
+
 
 class CaptchaBone(BaseBone):
     r"""
     The CaptchaBone is used to ensure that a user is not a bot.
 
     The Captcha bone uses the Google reCAPTCHA API to perform the Captcha
-    validation and is derived from the BaseBone.
+    validation and supports v2 and v3.
+
+    .. seealso::
+
+        Option :attr:`core.config.Security.captcha_default_credentials`
+        for global security settings.
 
-    :param publicKey: The public key for the Captcha validation.
-    :param privateKey: The private key for the Captcha validation.
-    :param \**kwargs: Additional arguments to pass to the base class constructor.
+        Option :attr:`core.config.Security.captcha_enforce_always`
+        for developing.
     """
+
     type = "captcha"
 
-    def __init__(self, *, publicKey=None, privateKey=None, **kwargs):
+    def __init__(
+        self,
+        *,
+        publicKey: str = None,
+        privateKey: str = None,
```

Comment:
Can you rename these to PEP8-compliant parameters and add a backward-compatibility?